.changeset/orange-buses-write.md

```diff
@@ -0,0 +1,15 @@
+---
+'astro': patch
+---
+
+Adds a new API to allow retrieving font files URLs when using the experimental fonts API
+
+You can now use the `getFontData()` function from `astro:assets` to have access to data for the provided CSS variable:
```

Comment:
```suggestion
The `getFontData()` helper function from `astro:assets` provides access to font family data for use outside of Astro. This can then be used in an [API Route](/en/guides/endpoints/#server-endpoints-api-routes) or to generate your own meta tags.
```